.travis.yml

```diff
@@ -3,6 +3,7 @@ language: node_js
 node_js:
   - "6.0"
 after_success:
+  - npm install -g codeclimate-test-reporter
```

Comment:
Could we just add this to the package.json as a devdep or something? Feels like an implicit dependency here.